benches/microbenchmarks/sort.py

```
@@ -0,0 +1,7 @@
+from random import random
+
+unsorted_list = [random() for _ in range(5 * ITERATIONS)]
```

Flake8 (7.2.0)

[error] undefined name 'ITERATIONS'

(F821)